assets/controllers/rich_textarea_controller.js

```diff
@@ -23,9 +25,12 @@ export default class extends Controller {
             this.toggleFormattingEnclosure('_');
         }
 
-        // grave to toggle inline code
-        else if (event.key === "`") {
-            this.toggleFormattingEnclosure('`');
+        // toggle wrapping on selection texts
+        else if (
+            this.element.selectionStart != this.element.selectionEnd
+            && wrapKeys.includes(event.key)
+        ) {
+            this.toggleFormattingEnclosure(event.key);
```

Comment:
both `*` and `_` evaluate to italic. this might be too crazy, but would it be possible to get doing `*` to add `**` on both sides instead so it adds bold? or should we not assume that was the users intent?

Reply:
or maybe it should just be additive always rather than this toggle add/remove? this is very nitpicky so I'm also fine with it as is if you think either aren't worthwhile

Reply:
also added the cycle thing, it's still rudimentary but it should work and not get in the way too much